sources/models/useful.js

```diff
@@ -0,0 +1,28 @@
+export const useful_functions = {
+
+	//For validatoin of inputs
+	titleValidation(value,rightLength){
+		var regExpResolt = value.search(/[\§\<\>@#\$\^&\*\+=\\~\[\]\{\}\|_]/g),
+				argLength = value.length;
+		if(regExpResolt === -1 && argLength >= 1 && argLength <= rightLength){
+			return value;
+		}else{
+			return false;
+		}
+	},
+
+	//For scrolling to the newly added elements
+	scrollToLastAddedElement(component,show,select){
+		var data_obj = component.data.pull;
+		var last_id = Object.keys(data_obj)[Object.keys(data_obj).length-1];
```

Comment:
it seems that it can be done simpler via the `component.getLastId()` method

Reply:
Ok